### PR TITLE
add instructions for ssh key gen and config

### PR DESCRIPTION
**Pull request by BRReed**
Signed-off-by: Brian R Reed <b.r.reed@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,4 +72,29 @@ Persistent settings
...
+
+    * Keep in mind this will overwrite your current key pair. Any existing devices that are key paired will no longer authenticate using the previous key.
+    4. Press Enter to accept no passphrase
+    5. Press Enter again to confirm no passphrase
+
+    * Your public key is now saved at `~/.ssh/id_rsa.pub`
+
+2. Copy your public SSH key to your server
+    1. Linux/MacOS: `ssh-copy-id -i ~/.ssh/id_rsa.pub <USERNAME>@<SERVER-IP>`
+    2. Windows: `cat ~/.ssh/id_rsa.pub | ssh <USERNAME>@<SERVER-IP> "cat >> ~/.ssh/authorized_keys"`
```
Comment (JoshRodstein) [inline README.md]: This step fails from **windows** to **raspios** due to directory & file `~/.ssh/authorized_keys` not existing on the pi.

... manually creating the path on the pi, and ree running the copy command from windows fixes the issue

**Review by JoshRodstein — CHANGES_REQUESTED** [review]
comment left and changes requested for **ssh key**s step 2.2

File: README.md [hunk 2]
```diff
@@ -72,4 +72,30 @@ Persistent settings
...
+    1. Check if you already have an ssh key pair (id_ras and id_rsa.pub) on your workstation. If you do skip to step 2 to copy the key to the server. Note: `~/` is a shortcut path to your home directory.
+    `ls -la ~/.ssh`
+    
+2. Create new ssh key pair.
+`ssh-keygen -t ed25519 -C "your_email@example.com"`
+    1. Press Enter to save the key at the default path.
+    2. Press Enter to accept no passphrase.
+    3. Press Enter again to confirm no passphrase.
+
+    * Your public key is now saved at `~/.ssh/id_ed25519.pub` and the private key is at `~/.sssh/id_ed25519`
```
Comment (JoshRodstein) [inline README.md]: type  "sssh" should be ".ssh"

File: README.md [hunk 3]
```diff
@@ -72,4 +72,30 @@ Persistent settings
...
+
+3. Copy your public SSH key to your server
+    1. Linux/MacOS: `ssh-copy-id -i ~/.ssh/id_ed25519.pub <USERNAME>@<SERVER-IP>`
+    2. Windows: `cat ~/.ssh/id_ed25519.pub | ssh <USERNAME>@<SERVER-IP> "cat > ~/.ssh/authorized_keys"`
```
Comment (stephenlauck) [inline README.md:97]: dir must exists before copying key. @BRReed going to fix.

**Review by stephenlauck — APPROVED** [review]

**Review by JoshRodstein — APPROVED** [review]

**Pull request merged**